ai-service/src/ai_service/main.py

```
@@ -1,27 +1,46 @@
-from ai_service.db import add_chunks, collection
+from ai_service import db
 from ai_service.embedder import embed_text
-
 from ai_service.ollama_client import chat_with_ollama
-
 from .exceptions import AIServiceError
 
 
+def upload_code(code: str):
```

Ruff (0.12.2)

Missing return type annotation for public function `upload_code`

Add return type annotation: `None`

(ANN201)